satellite/analytics/service.go

```diff
@@ -195,11 +391,26 @@ func (service *Service) TrackAccountVerified(userID uuid.UUID, email string) {
 		Event:      eventAccountVerified,
 		Properties: props,
 	})
+
+	dataEvent := map[string]interface{}{
+		"email":     email,
+		"eventName": "pe20293085_account_verified",
+		"properties": map[string]interface{}{
+			"userid": userID.String(),
+		},
+	}
+
+	go func() {
+		err := service.sendHubSpotEvent(service.ctx, &dataEvent)
+		if err != nil {
+			service.log.Error("Error sending Account Verified event", zap.Error(err))
+		}
+	}()
 }
 
 // TrackEvent sends an arbitrary event associated with user ID to Segment.
 // It is used for tracking occurrences of client-side events.
-func (service *Service) TrackEvent(eventName string, userID uuid.UUID) {
+func (service *Service) TrackEvent(ctx context.Context, eventName string, userID uuid.UUID, email string) {
```

Comment:
you no longer need to add the `ctx` parameter in here (and in `TrackCreateUser`). Because we are using service.ctx now